docs/guides/connecting_to_your_data/overview.md

```diff
@@ -0,0 +1,205 @@
+
+```console
+great_expectations datasource new
+```
+
+From there, you will be able to follow along a YAML based workflow for configuring and saving your Datasource.  Whether you prefer to work with the Jupyter Notebook's boilerplate for creating a datasource, or would rather dive in from scratch with a Python script, however, most of the work will take place in the configuring of the Datasource in question.
+
+
+### 2. Configure your Datasource
+
+Because the underlying data systems are different, configuration for each type of Datasource is slightly different.  We have step by step how-to guides that cover many common cases, and core concepts documentation to help you with more exotic kinds of configuration.  It is strongly advised that you find the guide that pertains to your use case and follow it.  If you are simply interested in learning about the process, however, the following will give you a broad overview of what you will be doing regardless of what your underlying data systems are.
+
+Datasource configurations can be written as YAML files or Python dictionaries.  Regardless of variations due to the underlying data systems, your Datasource's configuration will look roughly like this:
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python
+datasource_yaml = fr"""
+name: <name_of_your_datasource>
+class_name: Datasource
+execution_engine:
+    class_name: <class_of_execution_engine>
+data_connectors:
+    <name_of_your_data_connector>:
+        class_name: <class_of_data_connector>
+        <additional_keys_based_on_source_data_system>: <corresponding_values>
+"""
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python
+datasource_config = {
+    "name": "<name_of_your_datasource>",
+    "class_name": "Datasource",
+    "execution_engine": {"class_name": "<class_of_execution_engine>"},
+    "data_connectors": {
+        "<name_of_your_data_connector>": {
+            "class_name": "<class_of_data_connector>",
+            "<additional_keys_based_on_source_data_system>": "<corresponding_values>"
+        }
+    }
+}
+
+```
+
+</TabItem>
+</Tabs>
+
+Please note that this is just a broad outline of the configuration you will be making.  You will find much more detailed examples in our documentation on how to connect to specific source data systems.
+
+The `name` and `class_name` top level keys will be the first you need to define.  The `name` key can be anything you want, but it is best to use a descriptive name as you will use this to reference your Datasource in the future. Unless you are extending Great Expectations and using a subclass of Datasource, you will almost never need to use a `class_name` other than `Datasource` for the top level `class_name` value.
+
+#### Configuring your Datasource's Execution Engine
```

Comment:
I think this is too much detail for this guide---the details will depend on which type of DataSource you're using.